src/screens/Chat/TestUsers.js

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
This probably should go into a different file as it is not a screen (probably a new folder called temp or something)